prepare_input_files_lists/list_dataset_files.py

```diff
@@ -98,8 +127,8 @@ def __write_dataset_info(
     for i, files_list_batch in enumerate(files_list_batches):
         log.info(f"Processing batch {i+1}/{len(files_list_batches)}...")
         with concurrent.futures.ProcessPoolExecutor(max_workers=n_workers) as executor:
-            number_of_events += list(tqdm(executor.map(__get_number_of_events, files_list_batch), total=len(files_list_batch)))
-
+            #number_of_events += list(tqdm(executor.map(__get_number_of_events, files_list_batch, nano_aod), total=len(files_list_batch)))
```

Comment:
Can you remove the comment?
Also is the line below working for TreeMaker? I don't think so.